src/org/labkey/test/components/ui/grids/GridBar.java

```diff
@@ -331,6 +435,11 @@ protected MultiMenu findMenu(String buttonText)
 
             return menus.get(buttonText);
         }
+
+        protected final WebElement aliquotViewButton()
+        {
+            return Locator.tagWithAttributeContaining("button", "id", "aliquotviewselector").findWhenNeeded(this);
+        }
```

Comment:
Could probably use a regular bootstrap menu for this menu.
```suggestion

        protected final WebElement aliquotView = BootstrapMenu.finder(getDriver()).locatedBy(
                Locator.tagWithAttributeContaining("button", "id", "aliquotviewselector").parent()).findWhenNeeded(this);
```

Reply:
Updated.